packages/components/src/molecules/SkuSelector/SkuSelector.tsx

```diff
@@ -0,0 +1,160 @@
+import type { FunctionComponent, HTMLAttributes } from 'react'
+import React, { forwardRef } from 'react'
+import { Label, SROnly, Link, LinkProps, LinkElementType } from '../..'
+
+export interface SkuOption {
+  /**
+   * Alternative text description of the image.
+   */
+  alt?: string
+  /**
+   * Image URL.
+   */
+  src?: string
+  /**
+   * Label to describe the option when selected.
+   */
+  label: string
+  /**
+   * Current value for this option.
+   */
+  value: string
+  /**
+   * Specifies that this option should be disabled.
+   */
+  disabled?: boolean
+  /**
+   * Hex color code for this option. This is mandatory if you want to use the `Color` variant.
+   */
+  hexColor?: string
+}
+
+export interface SkuSelectorProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  /**
+   * ID of the current instance of the component.
+   */
+  id?: string
+  /**
+   * Specify which variant the component should handle.
+   */
+  variant: 'label' | 'image' | 'color'
+  /**
+   * SKU options that should be rendered.
+   */
+  options: SkuOption[]
+  /**
+   * Name of the SKU property that this selector is relative to.
+   */
+  skuPropertyName: string
+  /**
+   * Currently active variation's value.
+   */
+  activeVariations: Record<string, string>
+  /**
+   * Extends all Link Props.
+   */
+  linkProps?: Partial<LinkProps<LinkElementType>>
+  /**
+   * Function that mounts the href string.
+   */
+  mountItemHref: (option: SkuOption) => string
```

Comment:
Didn't notice the name before 😅 . But I would also rename it here. `mount` could be confusing.

Reply:
agreed!